### DISCUSSION TITLE
Exclusions Path in Windows Defender

### DISCUSSION THREAD

**Discussion by snehilshah** [JanDeDobbeleer/oh-my-posh#4659] [Troubleshoot] [answered]
After reading the docs on [Windows Exclusion](https://ohmyposh.dev/docs/faq#the-prompt-is-slow-delay-in-showing-the-prompt-between-commands) I have made the following exclusion rules. Can someone confirm if I have done it right please. 

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/83776946/d0794da1-d672-467e-8e29-de83f849e2b8)

The command `(Get-Command oh-my-posh).Source` gives the output, `C:\Users\Snehil\AppData\Local\Programs\oh-my-posh\bin\oh-my-posh.exe` which is what I have added to the path as *process* and also as a *file*.

I'm only running windows defender for my antivirus nothing else.

**Comment by JanDeDobbeleer** [comment, marked as answer]
Looks correct!
